pkgs/development/python-modules/python-sat/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildPythonPackage, fetchFromGitHub, lib, six, pypblib, pytestCheckHook }:
+
+buildPythonPackage rec {
+	pname = "python-sat";
+  version = "0.1.6.dev6";
```

Comment:
EditorConfig didn't catch that, but there's mixed indentation in this file. Please use spaces instead of tabs and 2 of them for 1 level of indent.